components/fonts/freefont/history

```diff
@@ -1 +1,2 @@
-system/font/freefont-ttf@0.20120503-2018.0.0.1 system/font/freefont
+system/font/freefont-ttf@0.20120503-2018.0.0.1 system/font/truetype/freefont
```

Comment:
`2018.0.0.2`?
```
FMRI: pkg://openindiana.org/system/font/freefont@0.20120503-2018.0.0.1:20181106T081715Z
```

Reply:
Same here... when this was republished only system/font/truetype/freefont@0.20120503-2018.0.0.1 provided content... but bumping as well for safety.